dvc/repo/experiments/utils.py

```diff
@@ -13,6 +23,35 @@
 )
 
 
+class AmbiguousExpRefInfo(InvalidArgumentError):
+    def __init__(
+        self,
+        exp_name: str,
+        exp_ref_list: Iterable[ExpRefInfo],
+        git_remote: Optional[str] = None,
+    ):
+        if git_remote:
+            msg = [
+                (
+                    f"Ambiguous name '{exp_name}' refers to multiple "
+                    "experiments. Use full refname to push one of the "
+                    "following:"
+                ),
+                "",
+            ]
+        else:
+            msg = [
+                (
+                    f"Ambiguous name '{exp_name}' refers to multiple "
+                    f"experiments in '{git_remote}'. Use full refname to pull "
+                    "one of the following:"
+                ),
+                "",
+            ]
```

Comment:
The error messages now are too specific, we don't know if we are pushing or pulling (and we could also be using `resolve_name` in something completely unrelated to push/pull). We should probably just use a more general message without `git_remote` for every case like:

`Ambiguous name '{exp_name}' refers to multiple experiments. Use one of the following full refnames instead:`